code/modules/mob/living/carbon/human/species_types/ipc.dm

```diff
@@ -15,6 +15,7 @@
 	default_features = list("mcolor" = "#7D7D7D", "ipc_screen" = "Static", "ipc_antenna" = "None", "ipc_chassis" = "Morpheus Cyberkinetics(Greyscale)")
 	meat = /obj/item/stack/sheet/plasteel{amount = 5}
 	skinned_type = /obj/item/stack/sheet/metal{amount = 10}
+	exotic_bloodtype = "Oil"
```

Comment:
Just a heads up, I'm 99% sure this won't actually do anything unless you remove `NOBLOOD` as well. Which would mean you would have to give them a heart as well. And they would need oil transfusions.

It gets pretty messy without `NOBLOOD`.

Reply:
This doesn't do anything but simply print the bloodtype on medical scanners and such as this. Could be set to something more fancy like `"null"` or `"<font color='red'>ERROR</font>"`, but I'm not creative enough to figure that out.